jobs/create-version-branch.js

```diff
@@ -34,26 +35,38 @@ module.exports = async function (
   if (!semver.validRange(oldVersion)) return
 
   const version = distTags[distTag]
-  const { installations, repositories } = await dbs()
+  const { installations, repositories, logs } = await dbs()
   const installation = await installations.get(accountId)
   const repository = await repositories.get(repositoryId)
-
+  const log = Log({logsDb: logs, accountId, repoSlug: repository.fullName, context: 'create-version-branch'})
+  log.info('started', {dependency, type, version, oldVersion})
   const satisfies = semver.satisfies(version, oldVersion)
   const lockFiles = ['package-lock.json', 'npm-shrinkwrap.json', 'yarn.lock']
   const hasLockFile = _.some(_.pick(repository.files, lockFiles))
-  if (satisfies && hasLockFile) return
+  if (satisfies && hasLockFile) {
+    log.info('exited: dependency satisfies semver & repository has a lockfile')
+    return
+  }
 
   const billing = await getActiveBilling(accountId)
-  if (repository.private && (!billing || await getAccountNeedsMarketplaceUpgrade(accountId))) return
+  if (repository.private && (!billing || await getAccountNeedsMarketplaceUpgrade(accountId))) {
+    log.warn('exited: payment required')
+    return
+  }
 
   const [owner, repo] = repository.fullName.split('/')
   const config = getConfig(repository)
-  if (_.includes(config.ignore, dependency)) return
+  if (_.includes(config.ignore, dependency)) {
+    log.warn('exited: dependency ignored')
+    return
+  }
   const installationId = installation.installation
   const ghqueue = githubQueue(installationId)
   const { default_branch: base } = await ghqueue.read(github => github.repos.get({ owner, repo }))
+  log.info('github: found default branch', {defaultBranch: base})
```

Comment:
maybe `using default branch`? “found” implies that the finding was the problem, which it isn’t in this case.